src/ngraph/op/convolution.cpp

```diff
@@ -584,13 +589,27 @@ Shape op::util::infer_convolution_output_shape(const Node* node,
         << "Output channel count for filters is zero (filters shape: " << filters_shape << ", "
         << "output channels on axis " << output_channel_axis_filters << ").";
 
-    NODE_VALIDATION_ASSERT(node, filters_shape[input_channel_axis_filters] == input_channel_count)
-        << "Input channel count for filters (" << filters_shape[input_channel_axis_filters] << ") "
-        << "does not match the number of channels in the data batch (" << input_channel_count
-        << ") "
-        << "(filter input shape: " << filters_shape << ", filter input channels on axis "
-        << input_channel_axis_filters << "; data batch shape: " << data_batch_shape
-        << ", data batch channels on axis " << batch_axis_data << ").";
+    if (node->get_friendly_name().find("GroupConvolution") != std::string::npos)
```

Comment:
I thought of using the new functions, but wanted to do that separately.

Reply:
Sounds good. In any case, discriminating on `get_friendly_name` is not what we want to do here, because the node name can be overridden whoever's constructing the graph. (Technically this is not likely to happen because the CPU passes themselves are the ones constructing this node, but I don't think we want to rely on it.) You _could_ use `description()` instead, which returns the class name.

In the long run I think the preferred fix would be to generalize the functions in `validation_util` to support group conv (without snooping on the node description), and switch things over to that.

Reply:
Thanks. I will use use get_name() as @rkimballn1  suggested for now.

Would it be better if I created another method for groupconv? infer_groupconvolution_output_shape() ?

Reply:
> Would it be better if I created another method for groupconv?
> infer_groupconvolution_output_shape() ?

The duplication would be unfortunate. I think we're going to refactor this later, so it's not a big deal.

@rkimballn1 Don't we need to use `description()` if we want to discriminate on the node class?

Reply:
@aprocter yes I agree. I did not look at the code bug yes, description is what you want here. Then you can get rid of the find thing and use a simple string compare.
`node->description() == "GroupConvolution"`